Tests/test_SeqIO.py

```diff
@@ -3172,6 +3172,18 @@ def test_genbank21(self):
             messages,
         )
 
+    def test_genbank22(self):
+        """Test that genbank format write doesn't destroy db_source in annotations"""
+        record = SeqIO.read('GenBank/CAA46787.gb', 'genbank')
```

Comment:
Could you use one of the existing test files, e.g. ``GenBank/protein_refseq.gb`` or ``dbsource_wrap.gb``?

Reply:
Excellent point. Will do.